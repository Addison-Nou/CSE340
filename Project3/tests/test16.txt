{
    a, b : INT;

    { 
    str1 : STRING;
    str1 = "1";

    bool1 : BOOLEAN;
    bool1 = TRUE;
    bool1 = <> + 3 1 + 2 2;

    long1 : REAL;
    long2 : REAL;
    long1 = + long2 + long1 
    long2;
    
    num1 : INT;
    num2 : INT;
    num1 = + * num2 num1 num2;
    
    realnum1 : REAL;
    realnum2 : REAL;
    realnum1 = + * realnum2 realnum1 realnum2;
    
    WHILE (> a b)
    {
        a = + a b;

    }
            a = + a a;
        a = + a a;



    }
}