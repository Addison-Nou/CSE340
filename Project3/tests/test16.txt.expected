UNINITIALIZED long2 14
UNINITIALIZED long1 14
UNINITIALIZED long2 15
UNINITIALIZED num2 19
UNINITIALIZED num1 19
UNINITIALIZED num2 19
UNINITIALIZED realnum2 23
UNINITIALIZED realnum1 23
UNINITIALIZED realnum2 23
UNINITIALIZED a 25
UNINITIALIZED b 25
UNINITIALIZED a 27
UNINITIALIZED b 27
UNINITIALIZED a 30
UNINITIALIZED a 30